src/sentry/api/endpoints/group_details.py

```diff
@@ -260,6 +256,30 @@ def get(self, request: Request, group) -> Response:
                 }
             )
 
+            participants = user_service.serialize_many(
+                filter={"user_ids": GroupSubscriptionManager.get_participating_user_ids(group)},
+                as_user=request.user,
+            )
+
+            for participant in participants:
+                participant["type"] = "user"
+
+            if features.has("organizations:team-workflow-notifications", group.organization):
+                team_ids = GroupSubscriptionManager.get_participating_team_ids(group)
+
+                teams = Team.objects.filter(id__in=team_ids)
+
+                serialized_teams = [
+                    serialize(team, request.user, TeamSerializer()) for team in teams
+                ]
+
+                for serialized_team in serialized_teams:
+                    serialized_team["type"] = "team"
```

Comment:
we can combine this into one pass over all the teams instead of two

```suggestion
                serialized_teams = []
                for team in teams:
                    serialized_team = serialize(team, request.user, TeamSerializer())
                    serialized_team["type"] = "team"
                    serialized_teams.append(serialized_team)
```

Reply:
type will right now, just be used for distinguishing whether a participant is a user/team so we can render it correctly on the front end.